.github/workflows/semantic-pr.yml

```diff
@@ -18,13 +18,7 @@ jobs:
         with:
           # Configure which types are allowed.
           # Default: https://github.com/commitizen/conventional-commit-types
-          types: |
-            fix
-            feat
-            doc
-            refactor
-            ops
-            chore
+          types: feat
```

Comment:
oops you should not change that ! Those are *all* the possible qualifiers to commit messages !